sections/04_study.md

```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
 probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim]. We
 anticipate that these steps will be replaced by deep learning, which will infer
 features simply by its ability to optimize a complex model against data.
+
+### Structure determination and cryo-electron microscopy
```

Comment:
The data types are very different, but this section seems like it would fit better if it immediately follows the protein structure prediction section.  That way you can include a transition from predicting structure to modeling structure with cryo-EM data.

Reply:
This section was moved and was re-worked to include a transition sentence.